static/js/publisher/builds/repoDisconnect.js

```diff
@@ -32,11 +38,11 @@ function initRepoDisconnect() {
       });
     });
 
-    repoDisconnectConfirm.addEventListener("click", e => {
-      e.preventDefault();
+    repoDisconnectConfirm.addEventListener("click", () => {
+      setTimeout(() => {
```

Comment:
Adding a setTimeout pulls this outside of the main context - so when the page starts loading and the main context is suspended, this will still trigger.